lib/modules/dosomething/dosomething_campaign/includes/Campaign.php

```diff
@@ -376,8 +376,12 @@ protected function getMobileAppDate() {
       $timing = [];
       $timing['dates'] = (dosomething_helpers_extract_field_data($this->node->field_mobile_app_date));
 
-      foreach ($timing['dates'] as $key => $date) {
-        $timing['dates'][$key] = dosomething_helpers_convert_date($date);
+      if ($timing['dates'] != null) {
```

Comment:
the method @angaither suggests, `isset()` will actually check if the variable is `null` or even exists. So it might be possible that the variable might not even exist (which is happening in the bug I'm working on right now), and `isset()` helps check for both scenarios :)